packages/runtime/src/metadataGeneration/tsoa.ts

```diff
@@ -42,6 +42,7 @@ export namespace Tsoa {
     default?: any;
     validators: Validators;
     deprecated: boolean;
+    exampleLabels?: string[];
```

Comment:
Change `example?: unknown[]` to `examples?: Record<string, unknown>` instead of adding labels separately.